system-int-test/test/cmr/system_int_test/search/collection_spatial_keyword_search_test.clj

```diff
@@ -31,7 +35,18 @@
            "LA" [coll4]
            ["Detroit"] [coll5 coll7]
            ["LL" "Detroit"] [coll5 coll6 coll7]
-           "BLAH" []))
+           "Gaza Strip" [coll-gaza]
+           "Tuolumne River Basin" [coll-iso-sample]
+           "North America" [coll-iso-sample]))
```

Comment:
Add a search for "Continent" finds all collections which have Continent at the top level.